deploy/.env.example

```diff
@@ -0,0 +1,42 @@
+# COMMON
+LOG_LEVEL=INFO
+NETWORK=mainnet
+ENABLE_HEALTH_SERVER='True'
+HEALTH_SERVER_PORT='8080'
+HEALTH_SERVER_HOST=0.0.0.0
+ETH1_CONFIRMATION_BLOCKS='15'
+
+# ORACLE
+IPFS_ENDPOINT='/dns/ipfs.infura.io/tcp/5001/https'
+IPFS_FETCH_ENDPOINTS='https://gateway.pinata.cloud,http://cloudflare-ipfs.com,https://ipfs.io'
+IPFS_PINATA_PIN_ENDPOINT='https://api.pinata.cloud/pinning/pinJSONToIPFS'
+IPFS_PINATA_API_KEY=''
+IPFS_PINATA_SECRET_KEY=''
+ETH2_ENDPOINT='http://localhost:3501'
+ORACLE_PRIVATE_KEY=0x<private_key>
+AWS_ACCESS_KEY_ID=''
+AWS_SECRET_ACCESS_KEY=''
+PROCESS_INTERVAL='180'
+
+# KEEPER
+WEB3_ENDPOINT=''
+KEEPER_PRIVATE_KEY=''
```

Comment:
For keeper the oracle private key is used as well

Reply:
ok, but .env files is common for all services so we don't need to specify it twice